src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -113,19 +113,19 @@
    :whiteboard/zoom-out          {:binding "shift+dash"
                                   :fn      #(.zoomOut (.-api ^js (state/active-tldraw-app)) false)}
 
-   :whiteboard/zoom-in           {:binding "shift+="
+   :whiteboard/zoom-in           {:binding "shift+equals"
```

Comment:
This is a nice readability improvement. If users use these special characters will their commands still work? If not, we should let the user know about the breaking change in the PR title